mock/service/controller.go

```diff
@@ -341,3 +341,23 @@ func (s *service) ControllerGetCapabilities(
 		},
```

Comment:
Why is `mock/mock` binary file checked in? Let's remove that.

Reply:
will remove